.github/ISSUE_TEMPLATE/bug_report.yaml

```diff
@@ -39,16 +39,32 @@ body:
     validations:
       required: true
 
-  - type: textarea
+  - type: dropdown
     id: environment
     attributes:
-      label: What operating system, browser and environment are you using?
-      value: |
-        - **Browser**: Firefox v109
-        - **OS**: Nixos v22.11
-        - **Env**: Local
-    validations:
-      required: true
+      label: On which environments the bug occurs?
+      multiple: true
+      options:
+        - Local
+        - Dev (DgexSol)
+        - Staging (DgexSol)
+        - Dev (SNCF)
+        - Recette (SNCF)
+        - Integration (SNCF)
+        - Other
```

Comment:
Maybe we could add two more options :
- Custom (Docker compose)
- Custom (Using Kubernetes Helm Chart)

Since we will soon have osedyne that will tighten some behavior with its execution environment it might be relevant?